landlab/plot/landslides/plot_landslides.py

```diff
@@ -0,0 +1,210 @@
+Created on Mon May 9 2020
+@author: christinab
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from landlab.plot import imshow_grid
+
+
+def test_field_initialized_to_range(ls_prob, input_var_names, unit_default_value):
+    """Testing if the output fields are initialized within range of default.
+    """
+    for name in input_var_names:
+
+        print(
+            "Input: (Min, Max) range of {value} model inputs: {pars}".format(
+                value=name, pars=unit_default_value[name]
+            )
+        )
+        print(
+            "Ouput: {value} default value initialized for each node:".format(value=name)
+        )
+        field = ls_prob.grid["node"][name]
+        print(field)
+
+        field_min = ls_prob.grid["node"][name].min()
+        field_max = ls_prob.grid["node"][name].max()
+        unit_min = unit_default_value[name][0]
+        unit_max = unit_default_value[name][1]
+
+        assert field_min >= unit_min
+        assert field_max <= unit_max
+
+        print("Test: {value} within range. Passes test.".format(value=name))
+        print("")
+
+
+def print_nodevalues(ls_prob):
+    """Print out the input range and resulting node values for 
+    all landslide component values on the node.
+    ----------
+    Input: Landlab landslide model instance  """
+
+    for name in ls_prob.grid["node"]:
+        field_min = round(ls_prob.grid["node"][name].min(), 3)
+        field_max = round(ls_prob.grid["node"][name].max(), 3)
+
+        print_minmax = print(
+            "{value} (Min, Max) range of ({value1},{value2}) values on node:".format(
+                value=name, value1=field_min, value2=field_max
+            )
+        )
+        print_value = print(
+            "All {value} default value initialized for each node:".format(value=name)
+        )
+        field = ls_prob.grid["node"][name]
+        print(field)
+
+        print("")
+
+
+def print_list_nodevalues(ls_prob, values_list):
+    """Print out the input range and resulting node values for 
+    all landslide component values on the node.
+    ----------
+    Input: Landlab landslide model instance  """
+
+    for name in values_list:
+        field_min = ls_prob.grid["node"][name].min()
+        field_max = ls_prob.grid["node"][name].max()
+
+        print_minmax = print(
+            "{value} (Min, Max) range of ({value1},{value2}) values on node:".format(
+                value=name, value1=field_min, value2=field_max
+            )
+        )
+        print_value = print(
+            "All {value} default value initialized for each node:".format(value=name)
+        )
+        field = ls_prob.grid["node"][name]
+        print(field)
+
+        print("")
+
+
+def plot_landslide_4variables(
+    littleplots,
+    inputgrid1,
+    inputgrid2,
+    inputgrid3,
+    inputgrid4,
+    variable,
+    maxval,
+    plot_subtitle,
+    cmap,
+    scalelabel,
+):
+    """Return figure with four subplots given landslide variables.
+    Parameters
```

Comment:
I'm somewhat confused at the point of this, so please forgive me if I'm being dense. This seems like a wrapper on imshow_grid to make a four panel plot. I think having something in landlab.plot that makes it easy to use imshowgrid to make multipanel plots could be really valuable, but it doesn't seem landslide specific to me. And rather than limiting to a four panel plot, could benefit from being more general. 

@mcflugen I'd be interested in your thoughts as well.